app/utils/draft.js

```diff
@@ -79,6 +83,70 @@ export function alertSendToGroups(formatMessage, notifyAllMessage, accept, cance
     );
 }
 
+export function tryOpenURL(url) {
+    Linking.canOpenURL(url).then((supported) => {
+        if (supported) {
+            Linking.openURL(url);
+        } else {
+            const {formatMessage} = this.context.intl;
+            Alert.alert(
+                formatMessage({
+                    id: 'mobile.server_link.error.title',
+                    defaultMessage: 'Link Error',
+                }),
+                formatMessage({
+                    id: 'mobile.server_link.error.text',
+                    defaultMessage: 'The link could not be found on this server.',
+                }),
+            );
+        }
+    });
+}
+
+export function errorBadChannel(intl) {
+    const message = {
+        id: t('mobile.server_link.unreachable_channel.error'),
```

Comment:
this one is also defined in post_list ad markdown_links, maybe is best to consolidate them